### PR TITLE
Update translations for CommCare release 2.29

### PR DESCRIPTION
**Pull request by phillipm**
In prep for the 2.29 release tomorrow

### CODE REVIEW

**Comment by phillipm** [comment]
Pinging @dimagi/product and part of HQ interrupt on this ( @NoahCarnahan )

**Comment by sheelio** [comment]
Looks good to me

File: messages_en-2.txt [hunk 1]
```diff
@@ -572,15 +574,19 @@ sync.fail.bad.network=Couldn't contact server. Please make sure an internet conn
 sync.fail.server.error=The server had an error processing your data. Please try again later. If the problem persists, contact technical support.
 sync.fail.unknown=An unexpected issue occurred during sync. Please try to sync again.
 
-sync.progress.title=Communicating with Server
```
Comment (czue) [inline messages_en-2.txt]: this is probably just revealing my ignorance of how this is used, but do we not need these old strings around to support legacy versions?

Comment (phillipm) [inline messages_en-2.txt]: I think these strings are used in 1 or 2 places: showing keys and default values in the excel translation file and populating `default/app_strings.txt` in cczs (I think..).
It makes a lot of sense for this file to be versioned and have HQ provide the correct version in those places, but I don't think that is something we have setup, so we always just serve the latest version. 

Could be wrong on all of this, it is one of those parts of the product that I'm not sure anyone fully understands...

**Comment by phillipm** [comment]
@NoahCarnahan might be nice to merge this before deploy today, since 2.29 has been released.

**Pull request merged**